frontend/test/__support__/store.ts

```diff
@@ -56,7 +48,42 @@ export interface EntitiesStateOpts {
   questions?: Card[];
 }
 
+const EntitiesSchema: Record<keyof EntitiesState, NormalizrSchema<any>> = {
+  actions: [ActionSchema],
+  alerts: [],
+  collections: [CollectionSchema],
+  dashboards: [DashboardSchema],
+  databases: [DatabaseSchema],
+  schemas: [SchemaSchema],
+  tables: [TableSchema],
+  fields: [FieldSchema],
+  metrics: [MetricSchema],
+  segments: [SegmentSchema],
+  snippets: [SnippetSchema],
+  users: [UserSchema],
+  questions: [QuestionSchema],
+};
+
+const EMPTY_STATE: EntitiesState = {
```

Comment:
@kulyk what if we exported `getPlaceholderEntitiesState` but make it not accept any parameters? This way we won't have to duplicate the list of empty objects here

Reply:
And we could leave `getPlaceholderEntitiesState` in its own `entities.ts` file in that case

Reply:
Oh, that's much better, good call 👍